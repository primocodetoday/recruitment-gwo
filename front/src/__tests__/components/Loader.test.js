import React from 'react';
import { render } from '@testing-library/react';
import { Loader } from 'components';

it('<Loader/> render properly', () => {
  const { getByText, container } = render(<Loader />);
  expect(getByText('Loading...')).toBeTruthy();
  expect(container.firstChild).toMatchSnapshot();
});
