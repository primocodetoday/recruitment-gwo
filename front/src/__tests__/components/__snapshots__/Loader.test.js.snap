// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader/> render properly 1`] = `
<div
  class="d-flex"
  style="height: 80vh;"
>
  <div
    class="mx-auto my-auto spinner-border"
    role="status"
  >
    <span
      class="sr-only"
    >
      Loading...
    </span>
  </div>
</div>
`;
